pyspecProcScripts/integral_w_error.py

```diff
@@ -85,7 +85,7 @@ def integral_w_errors(
         # mean divides by N₁ (indirect), integrate multiplies by Δf, and the
         # mean sums all elements (there are N₁N₂ elements)
         s_forerror -= s_forerror.C.mean_all_but([indirect, direct]).mean(direct)
-        s_forerror.run(lambda x: abs(x) ** 2 / 2).mean_all_but([direct, indirect]).mean(
+        s_forerror.run(lambda x: abs(x) ** 2).mean_all_but([direct, indirect]).mean(
```

Comment:
example `check_integration_error.py` now gives blue bars that don't match the others, and I think this is what breaks it.  You do want to divide by 2 here!

Reply:
added in the "/2"!